sdk/communication/tests.yml

```diff
@@ -59,7 +59,7 @@ stages:
       SKIP_LIVE_TEST: TRUE
       AZURE_SUBSCRIPTION_ID: $(acs-subscription-id)
       COMMUNICATION_LIVETEST_CONNECTION_STRING: $(communication-livetest-connection-string)
-      COMMUNICATION_PHONE_NUMBER: $(communication-livetest-phone-number)
+      COMMUNICATION_PHONE_NUMBER: $(java-communication-livetest-phone-number)
```

Comment:
FYI @sacheu

Reply:
Thanks @benbp . Java pipeline has been switched to use subscription configuration. This needs to be changed once this PR is rebased.